test/unit/stream-id-stack-tests.js

```diff
@@ -62,8 +63,8 @@ describe('StreamIdStack', function () {
     stack.clear();
   });
   it('should release unused groups', function (done) {
-    var releaseDelay = 50;
-    var stack = newInstance(3, releaseDelay - 10);
+    var releaseDelay = 100;
```

Comment:
The `osPrecision` can be made into a helper property and reused, as any value greater than 15.6 (1000 / 64) will work on modern Windows, Linux and macOS.
Any interval greater than twice the precision for the delay would work, ie: `delay ± precision`